specs/commandline/sign.md

```diff
@@ -36,15 +36,15 @@ Flags:
        --plain-http                 registry access via plain HTTP
        --plugin-config stringArray  {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
        --signature-format string    signature envelope format, options: "jws", "cose" (default "jws")
-       --signature-manifest string  manifest type for signature, options: "image", "artifact" (default "image")
+       --signature-manifest string  [Experimental] manifest type for signature, options: "image", "artifact" (default "image")
```

Comment:
We also need to update command help to reflect it as experimental.

We should call out that features/flags mark experimental can be modified and dropped in future. Also, Notation dont guaranty backward comparability from them.

Reply:
@priteshbandi  please check the update in below paragraph.